spk/lua/src/service-setup.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
```

Comment:
Please remove shebang and also PACKAGE and DNAME variables, which are useless (see `demoservice` or `mosquitto` as example...)